### PR TITLE
[Fix] Events should be blank for newly created organizations

### LINKED ISSUE #661: Events should be blank for newly created organizations
- **Issue Description:** Events should be blank for newly created organizations. Refer #537 
- **Issue Severity:** Medium
- **Environment Details:** Found in `talawa\lib\views\pages\events\events.dart`
- **Observed Behaviour:** Currently default data is getting loaded for newly created organizations.
- **Expected Behaviour:** It should be blank for newly created organizations
- **Steps to reproduce issue:** Create an organization and switch to the events tab.
- **Does this issue need immediate attention?** Yes
- **Are you willing to work on this issue:** Yes, I am willing to work on this issue!

### PR DESCRIPTION
**Pull request by Raunakk02**
**What kind of change does this PR introduce?**
Fixed the issue of all events being shown on events page.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**
Resolves #661 

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by palisadoes** [comment]
Please correct your code to pass our package analysis test. This needs to be resolved before we can review.

**Review by Sagar2366 — APPROVED** [review]

**Comment by Sagar2366** [comment]
@Raunakk02 button to create new event in this case is missing.

**Review by Sagar2366 — CHANGES_REQUESTED** [review]
Add button to create new event.

**Review by Sagar2366 — CHANGES_REQUESTED** [review]
Add button to create new event.

**Comment by Raunakk02** [comment]
I have created the logic to remove the button in the case when no organization is selected otherwise it shows if the current organization ID is not null.

**Comment by Raunakk02** [comment]
@Sagar2366 sir, I have fixed the issue now.

File: lib/views/pages/events/events.dart [hunk 1]
```diff
@@ -175,20 +181,24 @@ class _EventsState extends State<Events> {
...
+              int.parse(a['startTime'] as String))
           .compareTo(DateTime.fromMicrosecondsSinceEpoch(
-              int.parse(b['startTime'].toString())));
+              int.parse(b['startTime'] as String)));
     });
     eventsToDates(eventList, DateTime.now());
     setState(() {
       displayedEvents = eventList;
     });
+    // print('orgID ==== $currentOrgID');
```
Comment (Sagar2366) [inline lib/views/pages/events/events.dart:200]: remove code added for debugging.

Comment (Sagar2366) [inline lib/views/pages/events/events.dart:200]: remove code added for debugging.

Comment (Raunakk02) [reply lib/views/pages/events/events.dart]: I have removed the debug print code now @Sagar2366 sir.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**